docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
+    volumes:
+        - .:/app
```

Comment:
Can you explain a bit how a `/app/app/cache` and `/app/app/logs` volumes creation could improve performance on MacOS and Windows, please?